src/display/font_loader.js

```diff
@@ -367,14 +353,14 @@ var FontFaceObject = (function FontFaceObjectClosure() {
         return null;
       }
 
-      if (PDFJS.disableFontFace) {
+      if (this.options.disableFontFace) {
         this.disableFontFace = true;
         return null;
       }
 
       var nativeFontFace = new FontFace(this.loadedName, this.data, {});
 
-      if (PDFJS.pdfBug && 'FontInspector' in globalScope &&
+      if (getDefaultSetting('pdfBug') && 'FontInspector' in globalScope &&
```

Comment:
Since you're passing in both `isEvalSupported` and `disableFontFace` as options, are there any reason not do that with `pdfBug` as well?